roles/heat/tasks/main.yml

```diff
@@ -65,6 +65,14 @@
   run_once: true
   register: heat_domain
 
+- name: remove heat domain user from default domain
+  environment:
+    OS_IDENTITY_API_VERSION: 3
+    OS_DOMAIN_ID: default
+  shell: . /root/stackrc; openstack user delete --domain default heat_domain_admin
+  ignore_errors: True
```

Comment:
We don't use `ignore_errors` in Ursula. It puts task output in red if it would have been a failure. If we're going to ignore, we'd use `failed_when: False` instead.

However that's a pretty big hammer, I'd prefer to register the output and avoid failure if the text output has something to do with the user not existing, and otherwise fail for another normal failure reason.